src/libs/utils/hostosinfo.h

```diff
@@ -62,6 +64,7 @@ class QTCREATOR_UTILS_EXPORT HostOsInfo
     static HostArchitecture hostArchitecture();
 
     static constexpr bool isWindowsHost() { return hostOs() == OsTypeWindows; }
+    static constexpr bool isOs2Host() { return hostOs() == OsTypeOs2; }
```

Comment:
I would also add this:
```suggestion
    static constexpr bool isOs2Host() { return hostOs() == OsTypeOs2; }
    static constexpr bool isDosLikeHost() { return hostOs() == OsTypeWindows || hostOs() == OsTypeOs2; }
```
and use it everywhere instead of `isWindowsHost() || isOs2Host()`.